src/bokeh/document/document.py

```diff
@@ -754,11 +762,19 @@ def set_title(self, title: str, setter: Setter | None = None) -> None:
             self._title = title
             self.callbacks.trigger_on_change(TitleChangedEvent(self, title, setter))
 
-    def to_json(self, *, deferred: bool = True) -> DocJson:
+    @overload
+    def to_json(self, *, deferred: Literal[True] = ...) -> Serialized[DocJson]: ...
```

Comment:
Why `= ...`?

Reply:
This is a type only declaration, thus `...` should be used in place of real default values.